CONTRIBUTING.md

```diff
@@ -106,6 +108,20 @@ It is included in the `terra-cli.tar` install package in each GitHub release.
 This is the script users can add to their `$PATH` to invoke the CLI more easily from another directory.
 
 
+### Testing
+
+#### Override context directory
+The `.terra` context directory is stored in the user's home directory (`$HOME`) by default.
+You can override this default by setting the `TERRA_CONTEXT_DIR` environment variable to a valid directory.
+```
+export TERRA_CONTEXT_DIR="/Desktop/cli-testing"
```

Comment:
This is technically the parent of the context directory. It's not confusing enough that I'd necessarily say to change its behavior. Maybe name it `TERRA_CONTEXT_PARENT_DIR`  for clarity.

Reply:
Changed to `TERRA_CONTEXT_PARENT_DIR`